networkx/algorithms/dag.py

```diff
@@ -1223,37 +1224,86 @@ def dag_to_branching(G):
 
 @not_implemented_for("undirected")
 @nx._dispatchable
-def compute_v_structures(G):
-    """Iterate through the graph to compute all v-structures.
+def compute_colliders(G):
+    """In a DAG, if you have three nodes A, B, and C, and there are edges from A to C
+    and from B to C, then C is a collider. This means that both A and B are "causing" C
+    in the graph, and conditioning on C would open up an association between A and B.
+
+    Parameters
+    ----------
+    G : graph
+        A networkx `DiGraph`.
 
-    V-structures are triples in the directed graph where
-    two parent nodes point to the same child and the two parent nodes
-    are not adjacent.
+    Returns
```

Comment:
I guess we are using the `Yields` section heading instead of `Returns` now for generators docstrings.